ledfx/color.py

```
@@ -312,6 +376,48 @@
+    if raw_gradient:
+        config_string = raw_gradient
+    else:
+        # validate inline gradient string, will raise if invalid
+        validate_gradient(trimmed)
+        config_string = trimmed
+
+    # Resolve parsed gradient for sampling: prefer collection lookup
+    parsed = None
+    try:
+        parsed = gradients_collection[trimmed]
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)